deploy-config/sandbox.yml

```diff
@@ -9,5 +9,10 @@ admin_base_url: https://notify-sandbox.app.cloud.gov
 redis_enabled: 1
 default_toll_free_number: "+18885989205"
 ADMIN_CLIENT_SECRET: sandbox-notify-secret-key
+API_HOST_NAME: https://notify-api-sandbox.app.cloud.gov
 DANGEROUS_SALT: sandbox-notify-salt
```

Comment:
I went back and forth on this, and the secret key as well - it wasn't previously for the Sandbox, which is also an entirely disposable and throw-away environment.  Since multiple folks are going to be deploying things at any given time, trying to keep the salts in sync with the apps could become onerous for folks.

Perhaps there's a better way to handle this going forward; the sandbox environment is self-contained and disposable, like I said, but I definitely share the sentiment all the same.

Reply:
Put another way, I was optimizing for developer experience and usability here.  Do you or anyone else feel strongly otherwise?

@stvnrlly, do you think we should start managing some of these pieces the same as we do in other environments?

Reply:
Let's be consistent between environments as much as possible. If there's a reason to share the salt, we can use Drive and link to it from the docs

Reply:
That's a good idea - let's revisit as a follow-on task, thanks @stvnrlly!